cognite/client/data_classes/datapoints.py

```diff
@@ -235,6 +289,14 @@ def _extend(self, other_dps):
             else:
                 value.extend(other_value)
 
+    def _get_non_empty_data_keys(self):
```

Comment:
Can you not use the keys from `_get_non_empty_data_fields`?

Reply:
I was worried about the .copy, but then copied that as well -- and it's shallow anyway, right?